lib/ReactViews/Icon.jsx

```diff
@@ -141,13 +141,13 @@ export const StyledIcon = styled(Icon)`
   ${props => props.light && `fill: ${props.theme.textLight};`}
   ${props => props.dark && `fill: ${props.theme.textDark};`}
 
+  // Until we sort out what "light / dark" means for components that have both
+  // modes, use "realDark" to get real
+  ${props => props.realDark && `fill: ${props.theme.dark};`}
+
   ${props => props.fillColor && `fill: ${props.fillColor};`}
 
   ${props => props.opacity && `opacity: ${props.opacity};`}
 `;
 
-export default Icon;
-// (?) are these cjs exports for the doc generator?
-module.exports = Icon;
-module.exports.GLYPHS = GLYPHS;
-module.exports.StyledIcon = StyledIcon;
+export default Object.assign(Icon, { GLYPHS });
```

Comment:
Export things in only 1 way (using modules). This should fix the mismatch between Typescript type-checking and webpack imports. Ideally sometime we'd exports GLYPHS separately to Icon, but it's used so widely that that is best addressed in a separate PR